server/middlewares/index.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable import/prefer-default-export */
+import { decodeToken, displayMessage } from '<helpers>/utils';
+
+export const verifyUser = (req, res, next) => {
+  try {
+    const { token } = req.query;
+
+    const { id } = decodeToken(token);
+    req.userId = id;
+  } catch (error) {
+    return displayMessage(res, 500, error, null, false);
```

Comment:
🤔 I think the third argument on `displayMessage` is the response message, correct (^_^). Must be an oversight